docs/questions-and-answers.md

```diff
@@ -1,91 +1,58 @@
-<details><summary>What is the Open Economic Framework (OEF)?</summary>
-The <i>Open Economic Framework' (OEF)</i> consists of protocols, languages and market mechanisms agents use to search and find each other, communicate with as well as trade with each other. As such the OEF defines the decentralised virtual environment that supplies and supports APIs for autonomous third-party software agents, also known as Autonomous Economic Agents (AEAs).
+<details><summary>What is an AEA?</summary>
+AEA stands for Autonomous Economic Agent. AEAs act independently of constant user input and autonomously execute actions to achieve their objective. Their goal is to create economic value for you, their owner.
 <br><br>
-You can read more about the ledgers and the OEF <a href="../oef-ledger/"> here </a>
+You can read more about an introduction to AEAs in the introduction.
```

Comment:
I suggest:
`You can read an introduction to AEAs in <a>link<\a>.`

Reply:
There is an issue with this link working with our link checker. Not worth the effort.